test/e2e/network/netpol/probe.go

```diff
@@ -17,11 +17,17 @@ limitations under the License.
 package netpol
 
 import (
+	"fmt"
 	"github.com/onsi/ginkgo"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+// decouple us from k8smanager.go
```

Comment:
What's the point of doing this?  Now it's harder to click through the code because you hit the interface type instead of the implementation.

Reply:
we can just do a struct ? 
goal = decouple "Prober" bc its really confusing to see why the  "k8smanager"   is an input

Reply:
Can't figure out how to resolve this comment , but I'm fine with your changes here @jayunit100

Reply:
talked to matt on slack were fine w/ this interface